src/index.js

```diff
@@ -1436,8 +1529,8 @@ export default class ReactJkMusicPlayer extends PureComponent {
 
   onAudioVolumeChange = () => {
     const { volume } = this.audio
-    const { isVolumeFadeChang } = this.state
-    if (isVolumeFadeChang) {
+    const { currentVolumeFade } = this.state
+    if (currentVolumeFade !== VOLUME_FADE.NONE) {
```

Comment:
![image](https://user-images.githubusercontent.com/21015895/104436363-7930e600-55c8-11eb-8797-c50a1647739d.png)
![image](https://user-images.githubusercontent.com/21015895/104436398-83eb7b00-55c8-11eb-9862-f83af1a72055.png)

Now. toggle play, will be trigger `onAudioVolumeChange`  i think this is a bug

Reply:
Thanks for catching it